float32 wrist_extension
float32 lift_extension
---
bool success